tingbot/graphics.py

```diff
@@ -410,7 +395,7 @@ def _get_frames(self, pil_image):
             else:
                 palette = base_palette
 
-            pygame_image = pygame.image.fromstring(pil_image.tostring(), pil_image.size, pil_image.mode)
+            pygame_image = pygame.image.fromstring(pil_image.tobytes(), pil_image.size, pil_image.mode)
```

Comment:
The thinking was that we should prefer using a `bytes` data type rather than `str` in anticipation of transferring to python3. However, the tobytes function is only available in pillow (not PIL) which is what is actually installed on my machine. So I should probably revert this change for now.